src/deploy/campaign/environments.ts

```diff
@@ -146,8 +159,8 @@ export const validateEnv = (
   if (envLevel === "dev") return priceConfig;
 
   if (envLevel === "test" || envLevel === "dev") {
-    if (process.env.MOCK_MEOW_TOKEN === "false" && !process.env.STAKING_TOKEN_ADDRESS) {
-      throw new Error("Must provide a staking token address if not mocking MEOW token in `dev` environment");
+    if (!process.env.MOCK_Z_TOKEN && !process.env.STAKING_TOKEN_ADDRESS) {
```

Comment:
this is incorrect as well. at was correct before, why did you change it? see my previous comment about this variable